.github/workflows/ci.yml

```diff
@@ -0,0 +1,57 @@
+name: Continuous Integration
+on: [push, pull_request]
```

Comment:
Should this be just for the 'master' branch? That seems to be the default; not sure which is better.
There are also a number of other small differences compared to GitHub's default Ruby on Rails CI template. Can I make it more similar to the template?

Reply:
I'd prefer it to not be restricted to master, so I can push commits and have them tested without having to open a PR.

> Can I make it more similar to the template?

Sure.

Reply:
> I'd prefer it to not be restricted to master, so I can push commits and have them tested without having to open a PR.
Fair enough.